File: problem_solving_for_beginners/string_problems/python/str_comp.py
#!/usr/bin/python3
str1 = input("Enter first string: ")
str2 = input("Enter second string: ")

len1 = 0
len2 = 0
flag = 0

for i in str1:
    len1 += 1

for i in str2:
    len2 += 1

"""Compares two string"""
if len1 ==  len2:
    for i in range(len1):
        if str1[i] != str2[i]:
            flag += 1
            print("No")
            break
    else:
        print("Yes")
else:
    print("No")
        
       

File: problem_solving_for_beginners/string_problems/c-programming/str_comp.c
#include <stdio.h>

/**
 * _strcmp - Compares two strings
 * @str1: first string
 * @str2: seconf string
 *
 * Return: Print Yes otherwise
 */

void _strcmp(char *str1, char *str2)
{
    int idx = 0;

    while(str1[idx] != '\0' && str2[idx] != '\0' && str1[idx] == str2[idx])
    {
        idx++;
    }
    if (str1[idx] == '\0' && str2[idx] == '\0')
        printf("Yes\n");
    else
        printf("No\n");
}

int main(void)
{
    char str1[10], str2[10];
    printf("Enter two strings to compare: ");
    scanf("%s%s", str1, str2);

    //Write your code here
    _strcmp(str1, str2);

    return 0;
}